key_management.md

```diff
@@ -8,77 +8,96 @@ css_id: key_management
+On both the Director and the Image repository, the OEM maintains the keys to the Root, Timestamp, Snapshot, and Targets roles. However, for any delegated Targets roles on the Image repository, the corresponding keys are expected to be maintained by the delegated supplier. For example, If a tier-1 supplier signs its own images, then the supplier maintains its offline keys. Otherwise, the OEM would sign on behalf of the supplier, and thus maintain its keys.
 
 
 ### Online vs. offline keys
-Repository administrators SHOULD use offline keys to sign the root metadata on the director repository, so attackers cannot tamper with this file after a repository compromise. If a tier-1 suppliers signs its own images, then the supplier maintains its offline keys. Otherwise, the OEM would sign on behalf of the supplier, and thus maintain its keys.
 
-On the image repository, the timestamp, snapshot, and targets metadata SHOULD be signed using online keys, so that an automated process can instantly generate fresh metadata. On the image repository,
-there are two options for signing the timestamp and snapshot metadata, each with the opposite trade-off from the other. In the first option, the OEM uses online keys meaning automated processes can automatically renew the timestamp and snapshot metadata to indicate that there are new targets metadata and / or images available. However, attackers who have compromised a supplier’s key would now have access to the image repository and can instantly publish malicious images. If they additionally compromise the director repository, then they can execute arbitrary software attacks by selecting these malicious images on the image repository for installation. Such an attack could also facilitate mix-and-match attacks.
+Repository administrators SHOULD use offline keys to sign the Root metadata on the Director repository, so attackers cannot tamper with this file after a repository compromise. The Timestamp, Snapshot, and Targets metadata SHOULD be signed using online keys, so that an automated process can instantly generate fresh metadata.
+
+On the Image repository, the Timestamp, Snapshot, and Targets metadata SHOULD be signed using online keys, so that an automated process can instantly generate fresh metadata. There are two options for signing the Timestamp and Snapshot metadata, each with the opposite trade-off from the other. In the first option, the OEM uses online keys meaning automated processes can automatically renew the Timestamp and Snapshot metadata to indicate that there are new Targets metadata and/or images available. However, attackers who have compromised a supplier's key would now have access to the Image repository and can instantly publish malicious images. If they additionally compromise the Director repository, then they can execute arbitrary software attacks by selecting these malicious images on the Image repository for installation. Such an attack could also facilitate mix-and-match attacks.
```

Comment:
Targets metadata on the image repo should generally be offline-signed. This is correct in the subsequent paragraphs, so I think this was probably copy-pasted from the director section.

Reply:
Done. Yes, this was a copy-paste error. I've specified the Image keys roles that should be offline explicitly as well just to be clearer.